Godeps/_workspace/src/github.com/cloudflare/cfssl/crypto/pkcs11key/pkcs11key.go

```diff
@@ -159,35 +165,36 @@ func (ps *PKCS11Key) Destroy() {
 }
 
 func (ps *PKCS11Key) openSession() (session pkcs11.SessionHandle, err error) {
-	// Find slot by description
 	slots, err := ps.module.GetSlotList(true)
```

Comment:
this line may not appear necessary, but it is. Without calling GetSlotList, you get a `pkcs11: 0x3: CKR_SLOT_ID_INVALID` on the GetTokenInfo even though we discard the `slots` var.